examples/data-sources/nios_dhcp_shared_network/data-source.tf

```
@@ -0,0 +1,16 @@
+// Retrieve a specific SharedNetwork record by filters
```

```suggestion
// Retrieve a specific Shared Network record by filters
```
Please do this change throughout examples.